### PR TITLE
Multiple client blockstores 

### PR DESCRIPTION
**Pull request by magik6k**

### CODE REVIEW

File: node/impl/client/client.go [hunk 1]
```diff
@@ -348,13 +379,21 @@ func (a *API) ClientRetrieve(ctx context.Context, order api.RetrievalOrder, ref
...
 
-		order.MinerPeerID = peer.ID(mi.PeerId)
+		order.MinerPeerID = mi.PeerId
 	}
 
 	if order.Size == 0 {
 		return xerrors.Errorf("cannot make retrieval deal for zero bytes")
 	}
 
+	/*id, st, err := a.imgr().NewStore()
```
Comment (whyrusleeping) [inline node/impl/client/client.go:389]: deadcode?

File: node/impl/client/client.go [hunk 2]
```diff
@@ -399,7 +438,7 @@ func (a *API) ClientRetrieve(ctx context.Context, order api.RetrievalOrder, ref
 		order.Total,
 		order.MinerPeerID,
 		order.Client,
-		order.Miner)
+		order.Miner) // TODO: pass the store here   somehow
```
Comment (whyrusleeping) [inline node/impl/client/client.go:442]: do we want to complete this TODO?

File: node/impl/client/client.go (hunk 1 above)
Comment (magik6k) [reply node/impl/client/client.go:390]: For when retrieval gets this feature

File: node/impl/client/client.go (hunk 2 above)
Comment (magik6k) [reply node/impl/client/client.go:442]: Nope, depends on a bunch of markets work

**Pull request merged**